framework/kms-keyring.md

```diff
@@ -94,46 +99,35 @@ and MAY return a KMS Client that can make the following API calls in the given A
 If the client supplier cannot supply a client for the requested region,
 it MUST communicate that fact to the KMS keyring.
 
-### Key IDs
-
-Key IDs is a list of strings identifying KMS CMKs, in ARN format.
-This list identifies the CMKs to be used for data key encryption and decryption with this keyring.
+### Key Names
 
-Each Key ID MUST be one of the following:
+The key name identifies the [AWS KMS customer master keys (CMKs)](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id)
+that the AWS KMS keyring uses for data key encryption and decryption.
 
-- A CMK [alias](https://docs.aws.amazon.com/kms/latest/developerguide/programming-aliases.html) (e.g. "alias/MyCryptoKey")
-- A well-formed key ARN (e.g. arn:aws:kms:us-east-1:999999999999:key/01234567-89ab-cdef-fedc-ba9876543210)
-- A well-formed alias ARN (e.g. arn:aws:kms:us-east-1:999999999999:alias/MyCryptoKey)
+The AWS KMS keyring accepts all [key identifiers accepted by AWS KMS](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id) as key names.
 
-See [AWS Documentation](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#arn-syntax-kms).
-
-Note that only key IDs in the key ARN format will ever be used for decryption.
-This is because encrypted data keys constructed by the KMS keyring will always store the ID of the
-CMK used to encrypt it in key ARN format, and [OnDecrypt](#ondecrypt) checks the key ID against that
+Note that only key names in the [key ARN format](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html#arn-syntax-kms) will ever be used for decryption.
+This is because encrypted data keys constructed by the AWS KMS keyring will always store the ID of the
+CMK used to encrypt it in key ARN format, and [OnDecrypt](#ondecrypt) checks the key name against that
```

Comment:
```suggestion
CMK used to encrypt it in key ARN format and [OnDecrypt](#ondecrypt) checks the key name against that
```

Reply:
I think the comma here is actually correct since the two parts are each independent clauses. But the first sentence is a bit weird. Going to fix this.